Modules/Products/Feature/Modules/ProductsExportImportTest.php

```
@@ -16,252 +19,160 @@ class ProductsExportImportTest extends AbstractCompanyPanelTestCase
 
     #[Test]
     #[Group('export')]
-    public function export_products_downloads_csv_with_correct_data(): void
+    public function it_dispatches_csv_export_job_v2(): void
     {
         /* Arrange */
+        Queue::fake();
+        Storage::fake('local');
         $products = Product::factory()->for($this->company)->count(3)->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$products'. (undefined)

(UnusedLocalVariable)